### ISSUE TITLE
Duplicator Is Busy! Permanent bug until server restart.

### ISSUE THREAD

**Issue by OceanTheMan** [wiremod/advdupe2#118]
**This Bug has been a very big issue for me and others**
This is currently an issue because every once in a while the duplicator will not load anything until the next server restart I do not know why this occurs (Sometimes After a lag spike)

**I am able to**

- Copy Single Or Multiple Objects
- Paste Single Or Multiple Objects
- Everything Else Except (Look Below)

**I cannot**
- Load any of my previously saved files

**I have tried**
- Re-installing gmod
- Removing all my addons
- Sending the txt files of the dupes to my friends (Works For Them)
- It used to work for me in the past (Im not doing anything wrong)

Basically every time I try to load a dupe it says "Duplicator is busy!"
I am 99% sure this is a server side issue.

**Comment by Matthww** [comment]
I'm having the same issue

**Comment by thegrb93** [comment]
It's a net issue. It's waiting for an upload to finish but never finishes
